pkg/nodeinfo/attributes.go

```diff
@@ -28,16 +28,17 @@ var log = logf.Log.WithName("nodeinfo")
 
 // Node labels used by nodeinfo package.
 const (
-	NodeLabelOSName           = "feature.node.kubernetes.io/system-os_release.ID"
-	NodeLabelOSVer            = "feature.node.kubernetes.io/system-os_release.VERSION_ID"
-	NodeLabelKernelVerFull    = "feature.node.kubernetes.io/kernel-version.full"
-	NodeLabelHostname         = "kubernetes.io/hostname"
-	NodeLabelCPUArch          = "kubernetes.io/arch"
-	NodeLabelMlnxNIC          = "feature.node.kubernetes.io/pci-15b3.present"
-	NodeLabelNvGPU            = "nvidia.com/gpu.present"
-	NodeLabelWaitOFED         = "network.nvidia.com/operator.mofed.wait"
-	NodeLabelCudaVersionMajor = "nvidia.com/cuda.driver.major"
-	NodeLabelOSTreeVersion    = "feature.node.kubernetes.io/system-os_release.OSTREE_VERSION"
+	NodeLabelOSName            = "feature.node.kubernetes.io/system-os_release.ID"
+	NodeLabelOSVer             = "feature.node.kubernetes.io/system-os_release.VERSION_ID"
+	NodeLabelKernelVerFull     = "feature.node.kubernetes.io/kernel-version.full"
+	NodeLabelKernelVersionFull = "feature.node.kubernetes.io/kernel-version.full"
```

Comment:
Looks like we already had a constant for this - is this a duplicate?

Reply:
Thanks, removed